HelloWorld.java

```diff
@@ -1,4 +1,5 @@
 package proj7BittingCerratoCohenEllmer.bantam;
+//TODO: change to ScannerTestFile?
```

Comment:
I would consider this change, I only didnt do it in case people have something setup in their IDE for HelloWorld as an argument!

Reply:
Do it, it makes sense.